drops/product.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: default
+title: Product Drop
+has_children: false
+parent: Engine Drops
+permalink: /drops/product
+---
+
+{% include page_header.html %}
+
+### Drop Attributes
+
+- id
+- brand
+- name
+- slug
+- category
+- description
+- price
+- sku
+- meta_title
+- display_image_url
+- option_menu_data
+- variants_by_option_value
+- is_sellable?
+- product_seo_title
+- taxons
+- gift_card?
+- gift_card
+
+### Drop Methods
+
+The Product Drop allows you to access information regarding a specific product
+from your database. You'd generally use this drop in a product partial.
+
+#### display_image_alt
```

Comment:
Probably better to just not document the stuff that people shouldn't use. It also wouldn't be much work to remove references to this property from whatever theme uses it and then remove it from the drop class.